### PR TITLE
Update design documents

### PR DESCRIPTION
**Pull request by journey2w**
Get the latest design guide from TD.
Modify video layout and broken letters.
 - Set width or height for video.
 - Change "’" to  "'"
 - Change '“' and '”' to  '"'
 - Delete useless wearable/notification directory.

### CODE REVIEW

File: docs/application/design/introduction/styles/animations.md [hunk 1]
```diff
@@ -22,15 +22,15 @@ The concept of fluid animation design represents the look and feel of a gentle b
 
 Animations on the screen should have a consistent direction. Disorienting animations create visual noise and confuse users. Uniform animation effects that take place in a timely manner can maximize usability by enhancing understanding of what is occurring on the screen.
 
-> Avoid overusing animations. Add animations to tasks only if they are related to the task and can enhance users’ understanding of the task. Meaningless, irrelevant animations only add confusion.
+> Avoid overusing animations. Add animations to tasks only if they are related to the task and can enhance users' understanding of the task. Meaningless, irrelevant animations only add confusion.
```
Comment (mijin85cho) [inline docs/application/design/introduction/styles/animations.md]: It looks like `users'` should changed into `user's`.

Comment (mijin85cho) [reply docs/application/design/introduction/styles/animations.md]: Please ignore this review : https://english.stackexchange.com/questions/36657/user-s-guide-vs-users-guide.

File: docs/application/design/index.md [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
 
-Let’s begin by sharing our vision for Tizen app design and the visual language we use. These concepts will help you provide users with an enjoyable Tizen experience.
+Let's begin by sharing our vision for Tizen app design and the visual language we use. These concepts will help you provide users with an enjoyable Tizen experience.
  
 ## Visions
 
-Tizen is a platform built for the future. It was developed to foster openness, which is essential for creating a rich user experience on mobile devices and the devices they connect to. Tizen nourishes users’ curiosity, allows them to easily reach their goals, and lets them express their personality. Tizen is a platform designed to grow with users, enriching their daily life.
+Tizen is a platform built for the future. It was developed to foster openness, which is essential for creating a rich user experience on mobile devices and the devices they connect to. Tizen nourishes users' curiosity, allows them to easily reach their goals, and lets them express their personality. Tizen is a platform designed to grow with users, enriching their daily life.
```
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/design/index.md]: Please ignore this review : https://english.stackexchange.com/questions/36657/user-s-guide-vs-users-guide.

File: docs/application/design/mobile/2.3/TOC.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # [New in Tizen 2.3](mobile/2.3/new-tizen-2.3.md)
```
Comment (mijin85cho) [inline docs/application/design/mobile/2.3/TOC.md]: Broken links

File: docs/application/design/wearable/ui-components/list.md [hunk 4]
```diff
@@ -70,7 +67,7 @@ You can design list pages using one of the following formats. The default height
 
 
 | ![](media/ui_components_10.2.4_1-850x174_1.png) | ![](media/ui_components_10.2.4_1-850x174_2.png) | ![](media/ui_components_10.2.4_1-850x174_3.png) |
-|-----|-----|-----|
+| -- | -- | -- |
```
Comment (mijin85cho) [inline docs/application/design/wearable/ui-components/list.md]: simple Question : Why spaces are added here?

File: docs/application/design/wearable/ui-components/page-indicators.md [hunk 5]
```diff
@@ -4,26 +4,26 @@ Page indicators show the number of pages in a series and indicate which page is
...
 
 -   **Circular page indicators**
 
-    These indicators curve around the top edge of the screen when the pages are navigated by a rotary action and a swipe. They can form an arc of up to 120 degrees and are center-aligned.
+    Circular page indicators curve around the top edge of the screen when the pages can be navigated by rotary action and swiping. They form an center-aligned arc of up to 120 degrees.
 
-| Circular page indicators | |
-|-----|-----|
+| **Circular page indicators** in developer's guides | |
+| --- | --- |
```
Comment (mijin85cho) [inline docs/application/design/wearable/ui-components/page-indicators.md]: Simple Question : Why spaces are added here and on line39? (These spaces are removed by #133 ).

File: docs/application/design/wearable/ui-components/pages.md [hunk 6]
```diff
@@ -9,35 +9,35 @@ Pages are where all the content of an app is presented. At any moment, an app sh
...
 
 -   **Body**
 
-    The body of a page can contain text or image content.
+    The body of a page contains content including the main content, images, buttons, or text.
 
     -   **Text content**
 
-        Text content should have margins on either side. Without these, it’s hard to read the text on a round display. When the text doesn't fit on one screen, the page scrolls vertically so users can see the full text. Refer to [Scroll bar](scroll-bars.md) for more details.
+        Text content should have margins on either side. Without these, it's hard to read the text on a round display. When the text doesn't fit on one screen, the page scrolls vertically so users can see the full text. Refer to [Scroll bar](scroll-bar.md) for more details.
```
Comment (mijin85cho) [inline docs/application/design/wearable/ui-components/pages.md]: It looks like `scroll-bars.md` is correct. Would you pleaes check?

File: docs/application/design/mobile/2.3/TOC.md (hunk 3 above)
Comment (journey2w) [reply docs/application/design/mobile/2.3/TOC.md]: Done

File: docs/application/design/wearable/ui-components/list.md (hunk 4 above)
Comment (journey2w) [reply docs/application/design/wearable/ui-components/list.md]: No problem. | -- | and |----| are same.

File: docs/application/design/wearable/ui-components/page-indicators.md (hunk 5 above)
Comment (journey2w) [reply docs/application/design/wearable/ui-components/page-indicators.md]: No problem. | -- | and |----| are same.

File: docs/application/design/wearable/ui-components/pages.md (hunk 6 above)
Comment (journey2w) [reply docs/application/design/wearable/ui-components/pages.md]: done.

**Pull request merged**